react/Components/imageEdit.tsx

```diff
@@ -51,14 +51,18 @@ const ImageEdit: FC = () => {
             </div>
           </div>
         </Dropzone>
-        <Button
-          onClick={(e: SyntheticEvent) => {
-            e.preventDefault()
-            provider.setShowImage(true)
-          }}
-        >
-          {intl.formatMessage(commonModal.cancel)}
-        </Button>
+        {provider.file.result ? (
+          <Button
+            onClick={(e: SyntheticEvent) => {
+              e.preventDefault()
+              provider.setShowImage(true)
+            }}
+          >
+            {intl.formatMessage(commonModal.cancel)}
+          </Button>
+        ) : (
+          ''
```

Comment:
How about having an empty tag?
```suggestion
          <></>
```

Reply:
In this case I think is a better idea to use the `&&` operator!

```
        {provider.file.result && (
          <Button
            onClick={(e: SyntheticEvent) => {
              e.preventDefault()
              provider.setShowImage(true)
            }}
          >
            {intl.formatMessage(commonModal.cancel)}
          </Button>
        )
```

Reply:
That's better!!